content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,269 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
 remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
 accepting the down payment, trusting that the remaining amount will be paid later on.
 
-Initial down payment request
-============================
+Create invoices
+===============
 
 When a sales order is confirmed, the option to create an invoice becomes available, via the
 :guilabel:`Create Invoice` button, located in the upper-left corner of the sales order form. When
 clicked, a :guilabel:`Create invoices` pop-up appears.
 
+.. image:: down_payment/create-invoices-popup-form.png
+   :align: center
+   :alt: Create invoices pop-up form that appears in Odoo Sales.
+
 .. note::
-   Invoices are automatically created in drafts, so they can be reviewed before validation.
+   Invoices are automatically created as drafts, so they can be reviewed before validation.
 
 On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
 :guilabel:`Create Invoice` field:
 
 - :guilabel:`Regular invoice`
-   .. note::
-      If :guilabel:`Regular Invoice` is selected the other fields disappear, as they only pertain to
-      down payment configurations.
 - :guilabel:`Down payment (percentage)`
 - :guilabel:`Down payment (fixed amount)`
 
-In terms of a down payment, it can either be a fixed amount or a percentage of the total amount.
+.. note::
+   If :guilabel:`Regular Invoice` is selected, the other fields disappear, as they only pertain to
+   down payment configurations.
+
+   .. image:: down_payment/create-invoices-popup-form-regular-option.png
+      :align: center
+      :alt: The Regular Invoice option selected on the Create invoices pop-up form.
```

Comment:
To cut down on the number of images in this doc, I think that this one can be omitted
```suggestion
```